examples/using-graphql-typegen/.eslintrc.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  root: true,
+  parser: "@typescript-eslint/parser",
+  plugins: [
+    "@typescript-eslint"
+  ],
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
```

Comment:
```suggestion
```
you can remove this line as `@typescript-eslint/recommended` config already extends this config
<img width="469" alt="image" src="https://user-images.githubusercontent.com/7361780/169694464-6d3a3225-b439-4ecc-be71-4a03fd6f0be8.png">